pyproject.toml

```diff
@@ -11,6 +11,7 @@ maintainers = [
 ]
 
 dependencies = [
+  "numpy < 2",
```

Comment:
Please leave a comment for why we are doing this and when we can remove this limitation